thali/thalilogger.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var winston = require('winston');
+
+module.exports = function(){
+    return winston;
```

Comment:
Per the IM conversation I'm wondering if we should just throw in a rewriter that will add a tag element to the metadata with a tag value passed in on the require. In other words:

```Javascript
module.exports = function(tag) {
   if (!tag || typeof tag !== 'string' || tag.length < 3) {
      throw new Exception("All logging must have a tag that is at least 3 characters long!");
   }
   winston.addRewriter(function(level, msg, meta) {
      if (!meta.tag) {
         meta.tag = tag;
      }
      return meta;
   });
   return winston;
}
```

Or something like that.